znai-website-gen/src/main/java/org/testingisdocumenting/znai/extensions/latex/LatexInlinedCodePlugin.java

```diff
@@ -39,7 +39,7 @@ public InlinedCodePlugin create() {
 
     @Override
     public PluginResult process(ComponentsRegistry componentsRegistry, Path markupPath, PluginParams pluginParams) {
-        latex = pluginParams.getFreeParam();
+        latex = pluginParams.getOpts().getRequiredString("latex");
```

Comment:
what do you think of naming param `src` to avoid `latex` `latex`?

Reply:
done!